pkg/messages/format.go

```diff
@@ -159,7 +159,7 @@ func FormatBounds(boundsSlice []Bounds) string {
 				},
 			)
 		} else {
-			numMessages := bounds.LastOffset - bounds.FirstOffset
+			numMessages := bounds.LastOffset - bounds.FirstOffset + 1
```

Comment:
I'm not sure this is the fix. We see negative `duration` sometimes as well. I think there may be something wrong with how we assign `LastOffset` or `FirstOffset`

Reply:
Sorry I am conflating two different issues, this is correct!